ru/lessons/basics/collections.md

```diff
@@ -95,9 +95,9 @@ iex> File.read("path/to/unknown/file")
 {:error, :enoent}
 ```
 
-## Списки с ключами
+## Ключевые списки
```

Comment:
https://github.com/doomspork/elixir-school/blob/master/GLOSSARY.md#elixir-related-terms

> keyword list - список с ключами

That's current translation in glossary. Please change it there too.